Gemfile

```diff
@@ -33,6 +33,7 @@ end
 group :system_tests do
   gem "puppet-module-posix-system-r#{minor_version}", require: false, platforms: [:ruby]
   gem "puppet-module-win-system-r#{minor_version}",   require: false, platforms: [:mswin, :mingw, :x64_mingw]
+  gem "rspec-puppet-utils",                           require: false, platforms: [:ruby]
```

Comment:
What does this gem provide that cannot be done by normal rspec-puppet? I see it hasn't been under active development in many years so I am hesitant about adding it. Perhaps the additional features you need should be made part of rspec-puppet?

The change will also be overwritten during a pdk update. It should be included as part of the .sync.yml and a pdk update ran. See [here](https://github.com/puppetlabs/pdk-templates) for more info. Let me know if you have not used pdk before and I can point you to some additional resources.

Reply:
I don't know.  I'm trying to build an acceptance test for the automatic parameter lookup and I'm failing.   I'll remove the associated changes.